go/pkg/bertymessenger/service.go

```diff
@@ -208,17 +207,12 @@ func New(client bertyprotocol.ProtocolServiceClient, opts *Opts) (Service, error
 
 	// subscribe to contact groups
 	{
-		var contacts []Contact
-		err := svc.db.Find(&contacts).Error
+		contacts, err := svc.db.getContactsByState(Contact_OutgoingRequestSent)
 		if err != nil {
 			return nil, err
 		}
 		for _, c := range contacts {
-			if c.State != Contact_OutgoingRequestSent {
```

Comment:
why remove this ? without this check we will subscribe twice on accepted contacts and subscribe needlessly in other cases